src/main/webapp/app/shared/subject/subject.component.ts

```diff
@@ -297,8 +299,8 @@ export class SubjectComponent implements OnInit, OnDestroy, OnChanges {
             })
     }
 
-    filterChanged(text) {
-        this.filterTriggerUpdate$.next(text);
+    filterChanged(field: string, text: string) {
+        this.filterTriggerUpdate$.next(field + text);
```

Comment:
@blootsvoets I changed it in an easier way!
What do you think?

Reply:
Yes, that also works, LGTM.